### PR TITLE
Mobileapps 1302

### PR DESCRIPTION
**Pull request by ankitgoyalgloballogic**

### CODE REVIEW

File: ContentApp/BusinessLayer/Analytics /AnalyticsConstants.swift [hunk 1]
```diff
@@ -67,6 +67,7 @@ enum Event {
         case shareExtension = "page_view_share_extension"
         case transfers = "page_view_transfers"
         case taskTab = "page_view_tasks"
+        case taskDetailTab = "page_view_task_view"
```
Comment (gl-lovekesh) [inline ContentApp/BusinessLayer/Analytics /AnalyticsConstants.swift:70]: correct the name as taskDetailScreen

File: ContentApp/PresentationLayer/Tasks/Models/Task Comments/TaskCommentModel.swift [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+         message: String?) {
+        
+        self.created = created
+        self.createdBy = createdBy
+        self.commentID = commentID
+        self.message = message
+        self.messageDate = self.dateFrom(date: created)
+    }
+    
+    private func dateFrom(date: Date?) -> String? {
```
Comment (gl-lovekesh) [inline ContentApp/PresentationLayer/Tasks/Models/Task Comments/TaskCommentModel.swift:45]: move this to date extension.

**Review by gl-lovekesh — CHANGES_REQUESTED** [review]

File: ContentApp/BusinessLayer/Analytics /AnalyticsConstants.swift (hunk 1 above)
Comment (ankitgoyalgloballogic) [reply ContentApp/BusinessLayer/Analytics /AnalyticsConstants.swift]: Done

File: ContentApp/PresentationLayer/Tasks/Models/Task Comments/TaskCommentModel.swift (hunk 2 above)
Comment (ankitgoyalgloballogic) [reply ContentApp/PresentationLayer/Tasks/Models/Task Comments/TaskCommentModel.swift]: Done

**Pull request merged**